outputformatter.go

```diff
@@ -140,10 +127,11 @@ func (this *outputFormatter) removeParagraphsWithFewWords() {
 	if language == "" {
 		language = "en"
 	}
-	allNodes := this.topNode.Children()
+	allNodes := this.topNode.Find("*")
 	allNodes.Each(func(i int, s *goquery.Selection) {
 		sw := this.config.stopWords.stopWordsCount(language, s.Text())
-		if sw.wordCount < 5 && s.Find("object").Length() == 0 && s.Find("em").Length() == 0 {
```

Comment:
you have s.Find("object").Length()   twice in there.

Btw, this change is too aggressive, it strips parts of sentences.  Check Test_AbcNewsGoCom()